config/params/fork.go

```diff
@@ -0,0 +1,103 @@
+package params
+
+import (
+	"github.com/OffchainLabs/prysm/v6/consensus-types/primitives"
+	ethpb "github.com/OffchainLabs/prysm/v6/proto/prysm/v1alpha1"
+	"github.com/pkg/errors"
+)
+
+// DigestChangesAfter checks if an allotted fork is in the following epoch.
+func DigestChangesAfter(e primitives.Epoch) bool {
+	_, ok := BeaconConfig().networkSchedule.activatedAt(e + 1)
+	return ok
+}
+
+// ForkDigestFromEpoch retrieves the fork digest from the current schedule determined
+// by the provided epoch.
+func ForkDigestUsingConfig(epoch primitives.Epoch, cfg *BeaconChainConfig) [4]byte {
+	entry := cfg.networkSchedule.ForEpoch(epoch)
+	return entry.ForkDigest
+}
+
+func ForkDigest(epoch primitives.Epoch) [4]byte {
+	return ForkDigestUsingConfig(epoch, BeaconConfig())
+}
+
+func computeForkDataRoot(version [4]byte, root [32]byte) ([32]byte, error) {
+	r, err := (&ethpb.ForkData{
+		CurrentVersion:        version[:],
+		GenesisValidatorsRoot: root[:],
+	}).HashTreeRoot()
+	if err != nil {
+		return [32]byte{}, err
+	}
+	return r, nil
+}
+
+// Fork returns the fork version for the given epoch.
+func Fork(epoch primitives.Epoch) (*ethpb.Fork, error) {
+	cfg := BeaconConfig()
+	return ForkFromConfig(cfg, epoch), nil
+}
+
+func ForkFromConfig(cfg *BeaconChainConfig, epoch primitives.Epoch) *ethpb.Fork {
+	current := cfg.networkSchedule.ForEpoch(epoch)
+	previous := current
+	if current.Epoch > 0 {
+		previous = BeaconConfig().networkSchedule.ForEpoch(current.Epoch - 1)
+	}
+	return &ethpb.Fork{
+		PreviousVersion: previous.ForkVersion[:],
+		CurrentVersion:  current.ForkVersion[:],
+		Epoch:           current.Epoch,
+	}
+}
+
+// RetrieveForkDataFromDigest performs the inverse, where it tries to determine the fork version
```

Comment:
```suggestion
// ForkDataFromDigest performs the inverse, where it tries to determine the fork version
```
This comment in isolation is a bit odd - inverse to what?